README.md

```diff
@@ -70,7 +71,7 @@ Pulumi provider repositories have the following general structure:
 1. Download the dependencies:
 
     ```bash
-    cd provider && go mod tidy && popd
+    cd provider && go mod tidy && cd -
```

Comment:
Looks like `popd` is a built in shell command (TIL!).  Also didn't know about `cd -`.  Nice find!